domains/nmsderp.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A place for my small projects, and maybe a blog in the future.",
+    "repo": "https://github.com/nmsderp/nmsderp.github.io",
+    "owner": {
+        "username": "nmsderp",
+        "email": "nmsderp@gmail.com",
+        "twitter": "i don't have a twitter lol"
```

Comment:
if you dont have twitter then please remove the field. other then that all is good

Reply:
alr